tests/test_data/test_data_adresse_missing_plz.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
kannst du die adressen noch in einen eigenen ordner in den test-daten packen? sonst wird das schnell unübersichtlich

Reply:
ja, da war ich am Anfang unsicher ob ich das schon in Unterordner packen soll.
War mir nicht sicher wie viel es wird.
Setze ich direkt um

Reply:
done